openbb_terminal/cryptocurrency/cryptocurrency_helpers.py

```diff
@@ -885,14 +885,14 @@ def find(
     export : str
         Export dataframe data to csv,json,xlsx file
     """
+    coin = query
```

Comment:
It's related to crypto so `coin` makes more sense instead of `query`.. We also show `coin` in the menu [attached in image below].. Instead, we can drop `query` in argument and replace it with `coin` instead.. will that be better?

![image](https://user-images.githubusercontent.com/114890815/195961535-0d546e7a-020b-44bb-a55c-8462eb68d8de.png)

Reply:
After some thoughts updated the merge with `coin` as there are more places where we need to update `query` argument if we do so.. I can revert to `query` if needed..

Reply:
After some further thoughts using `query` for now..